src/style.css

```diff
@@ -1,6 +1,14 @@
+html {
+  position: relative;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  --color-blue: #00acdc;
```

Comment:
It is a good practice to specify variables in the `:root` block. 
It is also worth giving meaningful names to variables, because the color may change during the development process and then you will have to fix these variables in the entire code, not just in the variable definition. For example:
```
:root {
  --active-color: #00acdc;
}
```